website/app/routes/show.js

```diff
@@ -19,7 +19,16 @@ export default class ShowRoute extends Route {
     },
   };
 
+  modelCache = {};
+
   model(params) {
+    const extractComponentName = (path) => path?.split('/').pop();
+    const componentName = extractComponentName(params.path);
+    // If the model is cached, return it
+    if (this.modelCache[componentName]) {
```

Comment:
I _think_ you could safely use `params.path` here directly and avoid the (unlikely) future situation where we have something with the same name in, say, `components/` and `patterns/`. Or did you find a specific reason you wanted to split off just the last part?

Reply:
ahh good point, didn't think of that edge case.

Reply:
The path is already extracted below (line 33)